events/poland/poland.yaml

```diff
@@ -2,9 +2,9 @@
 events:
   - name: Bal folk workshop
     links:
-      - "https://www.facebook.com/events/963376321050850"
-    start_date: 2022-02-23
-    end_date: 2022-02-23
```

Comment:
It would be good to keep old events around for historical reference, so I've added this back in alongside the new one.